.github/pull_request_template.md

```diff
@@ -5,14 +5,12 @@ and are not aimed at stabilizing and preparing the Collector for the release wil
 
 _Delete this paragraph before submitting._
 
-**Description:** <Describe what has changed. 
-Ex. Fixing a bug - Describe the bug and how this fixes the issue.
-Ex. Adding a feature - Explain what this achieves.>
+**Description:** <Describe what has changed.>
+<!--Ex. Fixing a bug - Describe the bug and how this fixes the issue.
+Ex. Adding a feature - Explain what this achieves.-->
```

Comment:
Not sure about this, since users can leave it in the commit comment, the UI will not show us but will be merged in the final PR comment. Is that a problem?

Reply:
Interesting. I thought the UI will show us if we did not delete it since I have experienced it once when creating a PR with aws-otel-collector. IMO, if we asked the developers to delete it, then why should we keep it in the commit comment though?